_posts/2018-02-27-duplication-you-are-welcome.md

```diff
@@ -0,0 +1,68 @@
+---
+layout: post
+asset-type: post
+name: duplication-you-are-welcome 
+title: Duplication, you're welcome
+description: Thoughts and misunderstandings about duplication
+date: 2018-02-27 18:00:00 +00:00
+author: Raquel M Carmena
+image:
+   src: /assets/custom/img/blog/2018-02-27-duplication-you-are-welcome/duplication-header.jpg
+tags:
+- quality
+- duplication
+- metrics
+
+---
+
+There are models to describe the process of learning as <a href="http://wiki.c2.com/?DreyfusModel">Dreyfus model</a> or <a href="http://wiki.c2.com/?ShuHaRi" target="_blank">Shuhari</a>, but I'd explain another process when learning a new technical skill. It would be composed by 2 phases: 
+
+1. **Fascination**. When we learn a new concept that sounds really good to solve our problems and we apply it in any place we have an apportuninity. And we are so convinced about its advantages that it seems we have a radar to detect a place in which we can apply it. It's like the first stage at Shuhari, but we are absolutely fascinated and we don't believe in _another world_.
```

Comment:
1. _This is_ when we learn a new concept
2. Place a comma after problems
3. Lose "and" at the beginning of the second sentence, i.e. "We are so convinced..."